### PR TITLE
Modifies the adminhelp system to allow for servers to send a reminder message about policy when opening the dialouge

### PR DESCRIPTION
**Pull request by F-OS**
:tada: 
[Changelogs]:

:cl: F-OS
admin: Adminhelp now contains a warning message so players know it's not a "I got outrobusted send help" button, this is alterable with config changes.
/:cl:

[why]:

Based on what i've heard, there is a genuine issue of adminhelp becoming the go to for minor issues that should be resolved IC, and this has a chilling effect on roleplay because players live in fear of the bwoink, i'm suggesting this as a small solution to that problem. 

Original pr idea credited to kor in #21535

### CODE REVIEW

**Review by MrsTonedOne — CHANGES_REQUESTED** [review]
1: Please use the policy message config system for the message text. We have to keep downstreams who have different rules in mind, and the policy config system was created to make that easy. See drones for an example of a policy message.

2: `usr <<` doesn't work in this repo as we use a special browser based output window, the `to_chat` line above (line 488) is a good example of how to send text to a player's chat window.

**Comment by F-OS** [comment]
@MrStonedOne 
1. Not seeing the policy message system, please clarify.
2. Understood, let me fix it.

**Comment by MrStonedOne** [comment]
@Cyberboss 

how does one work with the `policies.txt` config in the repo located here: https://github.com/tgstation/tgstation/blob/master/config/policies.txt

**Comment by F-OS** [comment]
I think this should work.

**Comment by nichlas0010** [comment]
[this is already a thing](https://github.com/tgstation/tgstation/blob/575d0989449ca3eb6eb003a2b7297c2332c42622/code/modules/admin/verbs/adminhelp.dm#L367)

**Comment by F-OS** [comment]
@nichlas0010 This is mainly preemptive. Also, as it's config based, it can be changed so hosts and administrative staff can personalize it for their server.

File: code/modules/admin/verbs/adminhelp.dm [hunk 1]
```diff
@@ -487,6 +487,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	if(GLOB.say_disabled)	//This is here to try to identify lag problems
 		to_chat(usr, "<span class='danger'>Speech is currently admin-disabled.</span>")
 		return
+	to_chat(usr, config.policies["adminhelpnotification"])
```
Comment (nichlas0010) [inline code/modules/admin/verbs/adminhelp.dm]: they get told about the notification, after they've already sent their ticket, and they can't actually stop it?

Comment (F-OS) [reply code/modules/admin/verbs/adminhelp.dm]: eee let me fix that

**Comment by optimumtact** [comment]
this does nothing to change player behaviour and is just sad

**Comment by F-OS** [comment]
@optimumtact Good point.

**Pull request closed without merging**